### PR TITLE
Fix/redshift config

### PR DESCRIPTION
**Pull request by shriram2712**

### CODE REVIEW

File: terraform-aws-redshift-cluster/variables.tf [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
   type        = string
-  default     = null
-} */
+  default     = "ap-southeast-2"
+}
+
+variable "secondary_region" {
+  description = "The region to deploy the redis-cluster in"
+  type        = string
+  default     = "ap-south-1"
```
Comment (srlynch1) [inline terraform-aws-redshift-cluster/variables.tf]: this isn't a valid region is this used

File: terraform-aws-redshift-cluster/main.tf [hunk 2]
```diff
@@ -3,20 +3,342 @@
...
+  # Scheduled actions
+  create_scheduled_action_iam_role = true
+  scheduled_actions = {
+    pause = {
+      name          = "${local.name}-pause"
+      description   = "Pause cluster every night"
+      schedule      = "cron(0 22 * * ? *)"
+      pause_cluster = true
+    }
+    resize = {
```
Comment (srlynch1) [inline terraform-aws-redshift-cluster/main.tf]: delete resize

File: terraform-aws-redshift-cluster/variables.tf (hunk 1 above)
Comment (shriram2712) [reply terraform-aws-redshift-cluster/variables.tf]: This is the Mumbai region ap-south-1. I had overriden all this in my auto tf vars to make it different regions. i can change this to some other region. Wanted to retain sydney based configs as you had set in other folders

File: terraform-aws-redshift-cluster/main.tf (hunk 2 above)
Comment (shriram2712) [reply terraform-aws-redshift-cluster/main.tf]: deleted

**Pull request merged**